torch/fx/_symbolic_trace.py

```diff
@@ -182,7 +182,9 @@ class Tracer(TracerBase):
     process. The different behaviors that can be overridden are described
     in the docstrings of the methods on this class.
     """
-    def __init__(self, autowrap_modules: Tuple[ModuleType] = (math, ), enable_cpatching: bool = False) -> None:
+    def __init__(self, autowrap_modules: Tuple[ModuleType] = (math, ),
+                 enable_cpatching: bool = False,
+                 param_shapes_constant: bool = False) -> None:
```

Comment:
BTW we should also add a description of `param_shapes_constant` in the docstring below

Reply:
added